static/js/settings_lab.js

```diff
@@ -5,35 +5,34 @@ var exports = {};
 exports.set_up = function () {
     $("#ui-settings-status").hide();
 
-    $("#ui-settings").on("click", "input[name='change_settings']", function (e) {
+    $(".change-setting").change(function (e) {
```

Comment:
This selector should be more precise, e.g. `"#ui-settings .change-setting"`.  Otherwise, there's a lot of risk that this JS code will fire when making changes somewhere else in the app.

Reply:
I will keep this in my mind from now onwards.